### PR TITLE
add other metrics for slow queries

### PR DESCRIPTION
**Pull request by tk-sugar**
I added two metrics the following.
- total time for slow query per minutes
- average time for slow query per minutes

metrics

please review.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -53,12 +67,43 @@ func (m MongoDBSlowQueriesPlugin) FetchMetrics() (map[string]float64, error) {
...
+									},
+									"total_time": bson.M{
+											"$sum": "$millis",
+									},
+									"average_time": bson.M{
+											"$avg": "$millis",
+									},
+							},
+					},
+			}
```
Comment (rinmu) [inline main.go]: Please fix indent.

Comment (rinmu) [inline main.go]: Please fix indent.

File: main.go [hunk 2]
```diff
@@ -38,6 +38,20 @@ func (m MongoDBSlowQueriesPlugin) GraphDefinition() map[string]mp.Graphs {
...
+		"slow_queries_total_time": {
+			Label: "MongoDB Slow Queries",
+			Unit:  "integer",
+			Metrics: []mp.Metrics{
+				{Name: "total_time", Label: "Slow Queries Total Time"},
+			},
+		},
+		"slow_queries_average_time": {
+			Label: "MongoDB Slow Queries",
+			Unit:  "integer",
```
Comment (rinmu) [inline main.go]: `"float"` ?

File: main.go (hunk 1 above)
Comment (tk-sugar) [reply main.go]: fixed

File: main.go (hunk 2 above)
Comment (tk-sugar) [reply main.go]: fixed

**Comment by rinmu** [comment]
Thank you @tk-sugar !

**Pull request merged**